pkg/cli/README.md

```diff
@@ -0,0 +1,68 @@
+# CLI utility design guidelines
+
+## Guidelines for operator-level utilities
+
+- Be concise in the common case, verbose with errors
```

Comment:
super nit: full stop end some bullet points but not others

Reply:
Fixed.